### PR TITLE
Notifications

### PR DESCRIPTION
**Pull request by dorweitman**

### CODE REVIEW

File: src/app/components/Header/Notification.jsx [hunk 1]
```diff
@@ -81,6 +102,9 @@ class Notification extends React.Component {
       }
     });
   }
+
+  getNumOfUnSeenNotifs = () => this.state.notifications.filter(item => item.wasSeen === false).length;
```
Comment (ShragaUser) [inline src/app/components/Header/Notification.jsx]: refactor -> 
!item.wasSeen

File: src/server/routes/api.js [hunk 2]
```diff
@@ -77,6 +78,19 @@ const api = db => {
       });
   });
 
+
+  router.post("/notifications/changeWasSeen", (req, res) => {
+    const { _id } = req.body;
+    notifications
+      .findOneAndUpdate(
+        { _id: new ObjectID(_id) },
```
Comment (ShragaUser) [inline src/server/routes/api.js]: why new ObjectID -> is it necessary?

File: src/app/components/Header/Notification.jsx [hunk 3]
```diff
@@ -54,15 +57,33 @@ class Notification extends React.Component {
...
     const cardsById = this.props.cardsById;
   }
+
   goToBoard(notification) {
+    this.wasSeenHandler(notification);
     const { history } = this.props;
     history.push(`/b/${notification.boardId}`);
   }
 
+  wasSeenHandler(notification) {
```
Comment (ShragaUser) [inline src/app/components/Header/Notification.jsx:67]: const {_id} = notification.

File: src/app/components/Header/Notification.jsx [hunk 4]
```diff
@@ -54,15 +57,33 @@ class Notification extends React.Component {
...
+    fetch("/api/notifications/changeWasSeen", {
+      method: "POST",
+      body: JSON.stringify({ _id: notification._id }),
```
Comment (ShragaUser) [inline src/app/components/Header/Notification.jsx]: {_id}

**Review by ShragaUser — CHANGES_REQUESTED** [review]
some of these are questions.
some are changes to be made.

File: src/app/components/Header/Notification.jsx [hunk 5]
```diff
@@ -81,6 +103,9 @@ class Notification extends React.Component {
       }
     });
   }
+
+  getNumOfUnSeenNotifs = () => this.state.notifications.filter(item => !item.wasSeen).length;
```
Comment (ghost) [inline src/app/components/Header/Notification.jsx]: move it to inside the render method and defined it as normal variable and not function

**Review by ghost — APPROVED** [review]
goad end eproved!

**Review by ShragaUser — APPROVED** [review]
yas

**Pull request merged**